libs/partners/snowflake/pyproject.toml

```diff
@@ -0,0 +1,90 @@
+[tool.poetry]
+name = "langchain-snowflake"
+version = "0.1.0"
+description = "An integration package connecting Snowflake and LangChain"
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain/tree/master/libs/partners/snowflake"
+"Release Notes" = "https://github.com/langchain-ai/langchain/releases?q=tag%3A%22snowflake%3D%3D0%22&expanded=true"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.12"
```

Comment:
why isn't this compatible with python 3.12?

Reply:
unfortunately, a current limitation of the `snowflake.core` package: https://pypi.org/project/snowflake.core/

but checking with the team on why this is the case

Reply:
double checked and this is due to a dependency on Snowpark in `snowflake.core`. The next version of `snowflake.core` won't have this dependency